packages/protoc-gen-connect-query/tsconfig.build.json

```diff
@@ -2,8 +2,10 @@
   "files": ["src/protoc-gen-connect-query-plugin.ts"],
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "outDir": "dist/cjs",
+    "moduleResolution": "node",
     "module": "CommonJS",
-    "resolveJsonModule": true
+    "outDir": "dist/cjs",
+    "resolveJsonModule": true,
+    "skipLibCheck": true
```

Comment:
Do we really need to skip lib check with "node" version resolution?

Reply:
We're basically not making any change for the plugin because of https://github.com/bufbuild/protobuf-es/issues/586, which should not be an issue for users at all because all Node.js versions can run the CJS binary just fine, right?

Reply:
Yeah, my intention was to keep this one the same, probably just got lost in the shuffle back and forth. I'll keep this CJS entirely.

Reply:
And it turns out, `skipLibCheck` is not required. It probably got added when I was messing with other settings. Removed.